README.md

```diff
@@ -16,7 +16,8 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
+```
+Clearfix is a Javascript function that fixes borders not extending around floated objects within them by using clear:both.
```

Comment:
no, clearfix is used to force an element to contain its floated children. please reference http://learnlayout.com/clearfix.html